src/components/PoolChart/index.tsx

```diff
@@ -59,7 +59,7 @@ const PoolChart = ({ address }: PoolChartProps): JSX.Element => {
   const theme = useTheme()
 
   // update the width on a window resize
-  const ref = useRef<ResponsiveContainer>(null)
+  const ref = useRef(null)
```

Comment:
why delete type

Reply:
ah,   because eslint/prettier of node 17+ state that it is wrong type.  but for now, it's ok with node 16, 
=> I will revert that code.
Thanks for supporting me in that.